### ISSUE TITLE
Ability to reset settings?

### ISSUE THREAD

**Issue by bigchumpus** [Maassoft/ColorControl#94]
I goofed! Under the "LG Controller" tab, I accidentally deleted "mode2" in the list on the bottom of the app. I couldn't find a way to reset the settings and I also tried uninstalling and reinstalling ColorControl. As a total amateur, I'm not even sure what exactly I did by deleting "mode2" but I'm wondering if anyone could help copy the settings here and I can just manually add the input back in (or if someone knows how I can reset the app that would be great too).

**Comment by Maassoft** [comment]
mode2 just switches over to the "Expert (Dark room)" picture mode. If you didn't use it before I wouldn't worry about it. And if you have a 2021 series or newer TV it wouldn't probably work anyway because of the changed menu layout.
The best way to switch picture modes now is using the "Expert" button or via a custom preset. Create a new preset and add the step Actions->Picture Mode and select "expert2" for the "Expert (Dark room)" picture mode.

**Comment by bigchumpus** [comment]
Much appreciate the response. I'm using an 65" LG C1 for reference.

Please excuse my ignorance, but as someone who is using this software [from a guide](https://www.reddit.com/r/OLED_Gaming/comments/mbpiwy/lg_oled_gamingpc_monitor_recommended_settings/) and not even totally confident in what this software does, does the "mode2" only effect the software or does this impact the TV/remote in any way? It looks like the software allows me to use a digital remote and my intuition says that by deleting "mode2" I did not remove any fuctions from my remote.

**Comment by Maassoft** [comment]
Yes, you only removed a preset from this app, not from the TV/remote. Those presets are originally made by me to change settings on the TV (2019 series) while navigating its menu structure, but they are not really needed anymore because you can now instantly toggle/change settings using the Expert button or Actions menu under the "Add step" button.

**Comment by bigchumpus** [comment]
Perfect, thank you! Appreciate the app and the help!

**Comment by bloodindark** [comment]
i am wondering how can i reset all the settings and delete all changes/profiles that i made in colorcontrol  , like revert to the default state that the system was before using colorcontrol , because i have tried even uninstalling nvidia driver and color profiles are still present in colorcontrol , even if i deleted the colorcontrol folder and redownloaded it again fresh...it's not ok to reinstall windows in this situation

**Comment by bloodindark** [comment]
nevermind,  figured it out on my own, i found the registry keys and also the location where the app saves files.
If you want to revet to default like you never used this app before , do this:
in regedit , Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\nvlddmkm\State\ here you have to see what folder (it surely is one without the Connector in name) is your device , but delete inside the folder the folowing 2 : colorformatconfig and ditherregistrykey.
go to this location : C:\Users\YOUUSER\AppData\Roaming and delete the Maassoft folder
